### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by tuantrieu1663** [wanghaisheng/tiktoka-studio-uploader#77] [General]
Hello, I really like your idea. I downloaded it and couldn't find the exe file in the video. How to start it?
Is it possible to post videos to tiktok at the present time?

**Comment by wanghaisheng** [comment]
I have not integrate tiktok to gui version.
I can send you early bird version of gui to test against youtube now
admin@tiktokastudio.com
there is no exe in this lib for you to run. it is all python code.
